source/faq/fundamentals.txt

```diff
@@ -41,35 +41,51 @@ the columns in a relational database table.
 How do I create a database and a collection?
 --------------------------------------------
 
+.. note:: 
+
+   You can enter the commands referenced in this FAQ by using 
+   the :doc:`Mongo Shell </mongo/>`. The MongoDB Shell is an 
```

Comment:
If we're referring to the legacy shell, "mongo" should be lower case.

Reply:
We may want to use

```
:binary:`~bin.mongo`
```

to link to the legacy shell.

Reply:
Thats what I was refering to in my comments earlier, that link was not rendering a valid link in 4.2 or 4.0. I did think about adding a :4.2: to conf.py (currently there is not one). But for some reason the binary was not working in that branch.